parsec/backend/postgresql/organization.py

```diff
@@ -99,14 +99,32 @@
 """
 )
 
+_q_check_users_limit = Q(
+    f"""
+    SELECT
+        organization._id
+    FROM organization
+    LEFT JOIN
+        user_ on user_.organization = organization._id
+    WHERE
+        organization = { q_organization_internal_id("$organization_id") }
+    GROUP BY organization._id
+    HAVING users_limit is NULL OR count(user_) < users_limit;
+"""
```

Comment:
The point of this query is to return a whether we've reached the user limit or not, I think might be able to simplify this into:
```SQL
SELECT
    organization.users_limit is NULL
    OR (
        SELECT count(*) FROM user_ WHERE user_.organization = organization._id
    ) < organization.users_limit
FROM organization
WHERE organization.organization_id = $organization_id
```

(I didn't try this query, so I might be completely broken 😄 )

Reply:
@touilleMan I understand your comment about the aim of this query but I don't know if it's simpler to use one subquery instead of one HAVING condition.